tests/regression/test_network_of_populations.py

```diff
@@ -21,6 +21,8 @@ def _assert_baseline(result, force_update=False):
 
 def _assert_baseline_dataframe(result, force_update=False):
     with tempfile.NamedTemporaryFile(mode="w") as fp:
```

Comment:
I know this is not changed by this PR, but could you add delete=False please for Windows compatibility?